content/start_pes.tex

```diff
@@ -1,7 +1,7 @@
 \apisummary{ 
     Called at the beginning of an \openshmem program to initialize the execution
     environment. This routine is deprecated and is provided for backwards
-    compatibility. Implementations must include it, and the routine should
+    compatibility. Implementations must include it. The routine should
```

Comment:
The original text is a correct use of _Comma Use No. 1_ in [Purdue's Extended Rules for Using Commas](https://owl.english.purdue.edu/owl/resource/607/02/).